f600233d1b154d507b782f84604bcb12/update/sys_script_include_8a897895dbef9d14b1227ea5f3961968.xml

```diff
@@ -11,68 +11,100 @@ Payload is sent to the HOTH API for document generation.</description>
-						standardSpecifications: evalPlan.standard_specifications ? evalPlan.standard_specifications.toString().split(",").map(function(spec) { 
-							var record = new GlideRecord("x_g_dis_atat_eval_plan_assessment_area");
-							record.get(spec);
-							return record.name.toString();}) : null,
-						customSpecifications: evalPlan.custom_specifications ?evalPlan.custom_specifications.toString().split(",").map(function(spec) { 						
-							return spec;}) : null,
-						standardDifferentiators:  evalPlan.standard_differentiators ? evalPlan.standard_differentiators.toString().split(",").map(function(diff) { 
-							var record = new GlideRecord("x_g_dis_atat_eval_plan_differentiator");
-							record.get(diff);
-							return record.name.toString();}) : null,
-						customDifferentiators: evalPlan.custom_differentiators ? evalPlan.custom_differentiators.toString().split(",").map(function(diff) { 						
-							return diff;}) : null
-					}
-				};
-			}
-			else {
-				throw this.errUtil.createError("No Eval Plan found for this Acq Package. ", this.errUtil.INVALID_INPUT);
-			}
-		} catch(err) {
-			throw this.errUtil.createError("Error getting Eval Plan payload. " + err, this.errUtil.INVALID_INPUT);
-		}
-	},
+    getPayload: function() {
+        try {
+            var evalPlan = this.acquisitionPackage.evaluation_plan;
+            var projectOverview = this.acquisitionPackage.project_overview;
+            var EVAL_PLAN_ASSESSMENT_AREA_TABLE = "x_g_dis_atat_eval_plan_assessment_area";
+            var EVAL_PLAN_DIFFERENTIATOR_TABLE = "x_g_dis_atat_eval_plan_differentiator";
+            var includeCustomSpecificationsFlag = false; // encompasses both compliance standards, and assessment areas
+            var includeCustomDifferentiatorsFlag = false;
+            if (evalPlan) {
+                return {
+                    documentType: "EVALUATION_PLAN",
+                    templatePayload: {
+                        taskOrderTitle: projectOverview.title ? projectOverview.title.toString() : null,
+                        sourceSelection: evalPlan.source_selection ? evalPlan.source_selection.toString() : null,
+                        method: evalPlan.method ? evalPlan.method.toString() : null,
 
-	type: 'EvalPlan'
+                        standardSpecifications: evalPlan.standard_specifications ? evalPlan.standard_specifications.toString().split(",")
+                            .filter(function(recordSysId) {
+                                var record = new GlideRecord(EVAL_PLAN_ASSESSMENT_AREA_TABLE);
+                                record.get(recordSysId);
+                                if (record.name.toString() === "CustomAssessment") { // is this the special row in the ref table?
+                                    includeCustomSpecificationsFlag = true; // include customs later
+                                    return false; // filter out this option
+                                }
```

Comment:
Understood.  Tony said that `has_custom_specifications` is used, but then we don't have a corresponding `has_custom_differentiators`.  So this may need a bit more testing, but I'm going to merge this as is.  Still will need the filter()s so we don't up displaying the custom indicators with the optional items.